lib/portfinder.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * portfinder.js typescript definitions.
+ *
+ * (C) 2011, Charlie Robbins
+ */
+
+declare type PortfinderHandler = (err: Error, port: number) => void;
+
+/**
+ * The lowest port to begin any port search from.
+ */
+export declare let basePort: number;
+
+/**
+ * Responds with a unbound port on the current machine.
+ */
+export declare function getPort(callback: PortfinderHandler): void;
```

Comment:
@jucrouzet - thanks a bunch - `getPort` actually takes an optional options object and a callback. The options are `host: string` and `port: number` - do u mind adding those? (https://github.com/indexzero/node-portfinder/blob/c5dd2cc76bd26d4771cee9e0637dfa27a7f89f24/lib/portfinder.js#L25-L28)

also - if u pass a port option, it takes precedence over the `exports.basePort` setting - https://github.com/indexzero/node-portfinder/blob/c5dd2cc76bd26d4771cee9e0637dfa27a7f89f24/lib/portfinder.js#L30
(not sure how u declare that in TS, if not possible, no worries)

mostly, do u mind adding the options object?

then i'll merge this in, thanks again